Makefile

```
@@ -0,0 +1,40 @@
+﻿.PHONY: help install install-dev format check lint test test-fast clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)